src/sas/qtgui/MainWindow/DataExplorer.py

```diff
@@ -153,6 +153,16 @@ def __init__(self, parent=None, guimanager=None, manager=None):
         # Current view on model
         self.current_view = self.treeView
 
+    @property
+    def default_load_location(self) -> str:
+        return self._default_load_location
+
+    @default_load_location.setter
```

Comment:
Is there are reason why you've made a copy of the folder, and trying to synchronise two variables.

Seems to be against the general principle of storing things in only one place?

Reply:
Good catch. This could leave to divergence in the two variables if we aren't careful. Updated.

Reply:
Yup